server/tinylicious/package.json

```diff
@@ -74,19 +75,18 @@
     "@types/detect-port": "^1.3.0",
     "@types/json-stringify-safe": "^5.0.0",
     "@types/jsonwebtoken": "^8.3.7",
-    "@types/lodash": "^4.14.149",
+    "@types/lodash": "^4.14.118",
     "@types/mocha": "^5.2.5",
     "@types/morgan": "^1.7.35",
     "@types/nconf": "^0.10.0",
-    "@types/node": "^10.17.13",
+    "@types/node": "^12.19.0",
```

Comment:
Required to avoid errors when compiling .d.ts of latest server dependencies (which is why I was diffing package.json files.)